packages/fxa-settings/src/lib/oauth/oauth-errors.ts

```diff
@@ -202,6 +204,7 @@ export class OAuthError extends Error {
     super(msg);
     this.name = 'OAuthError';
     this.errno = err?.errno || OAUTH_ERRORS.UNEXPECTED_ERROR.errno;
+    this.response_error_code = err?.response_error_code;
```

Comment:
Probably the most crucial bit of this PR.  The RPs need this value in the params.